cypress/e2e/user.cy.js

```diff
@@ -1,12 +1,88 @@
 /// <reference types='cypress' />
 /// <reference types='../support' />
+import HomePageObject from '../support/pages/home.pageObject';
+import SettingsPageObject from '../support/pages/settings.pageObject';
+import SignUpPage from '../support/pages/signUp.pageObject';
+import ArticlePage from '../support/pages/article.pageObject';
+import faker from 'faker';
+import { confirmationMessage } from '../plugins/alertMessages';
+
+const settingsPage = new SettingsPageObject();
+const homePage = new HomePageObject();
+const articlePage = new ArticlePage();
 
 describe('User', () => {
+  let user;
+  let title;
+  let description;
+  let content;
+  let tags;
+  let nameNextUser;
+  let emailNextUser;
+  let passwordNextUser;
+  let followButtonSelector;
+  let followButtonText;
+  let unfollowButtonSelector;
+  let unfollowButtonText;
+  const signUpPage = new SignUpPage();
+
   before(() => {
+    cy.task('db:clear');
+    cy.task('generateUser').then((generateUser) => {
+      user = generateUser;
+
+      cy.visit('/#/register');
+      signUpPage.fillForm(user.username, user.email, user.password);
+      signUpPage.submitSignUpForm();
+
+      cy.contains(confirmationMessage).click();
+      homePage.clickNewArticle();
+
+      title = faker.lorem.words(2);
+      description = faker.lorem.words(2);
+      content = faker.lorem.words(2);
+      tags = 'tag1';
+
+      articlePage.fillTitle(title);
+      articlePage.fillDescription(description);
+      articlePage.fillContent(content);
+      articlePage.fillTags(tags);
+      articlePage.submitArticle();
 
+      homePage.clickSettings();
+      cy.scrollTo('bottom');
+      settingsPage.logOutSettings();
+    });
   });
 
-  it.skip('should be able to follow the another user', () => {
+  beforeEach(() => {
+    nameNextUser = faker.name.firstName();
+    emailNextUser = faker.internet.email();
+    passwordNextUser = faker.internet.password();
+
+    cy.visit('/#/register');
+    signUpPage.fillForm(nameNextUser, emailNextUser, passwordNextUser);
+    signUpPage.submitSignUpForm(); cy.contains(confirmationMessage).click();
```

Comment:
The methods 'submitSignUpForm()' and 'cy.contains(confirmationMessage).click()' should be on separate lines for better readability.

Reply:
Done